scripts/ChangesSinceLastRelease.txt

```diff
@@ -414,6 +414,121 @@ Windows.Win32.UI.Controls.RichEdit.CHARFORMATA.bPitchAndFamily...Windows.Win32.G
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINELEFT removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINERIGHT removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINEUP removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGEDOWN removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGELEFT removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGERIGHT removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGEUP removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_RIGHT removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_THUMBPOSITION removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_THUMBTRACK removed
+Windows.Win32.UI.WindowsAndMessaging.Apis.SB_TOP removed
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_BOTTOM added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_ENDSCROLL added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LEFT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINEDOWN added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINELEFT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINERIGHT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINEUP added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGEDOWN added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGELEFT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGERIGHT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGEUP added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_RIGHT added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_THUMBPOSITION added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_THUMBTRACK added
+Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_TOP added
+# OleCtl.h functions 
+Windows.Win32.System.Ole.Apis.OleCreateFontIndirect : lpFontDesc : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreateFontIndirect : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleCreatePictureIndirect : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleCreatePictureIndirect : lpPictDesc : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pPageClsID : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : ppUnk : [Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pvReserved : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrameIndirect : lpParams : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleLoadPicture : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleLoadPictureEx : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleLoadPicturePath : ppvRet : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleTranslateColor : lpcolorref : [In,Out] => [Out]
 # UIA_ enums
 Windows.Win32.System.SystemServices.UIA_ANNOTATIONTYPE added
```

Comment:
APIs in the SystemServices namespace indicates a bug. Even though the UIA changes went in with another PR, please update enums.json to move anything in SystemServices over to a more relevant namespace like Accessibility.

Either add a usage so the enum follows the namespace of the APIs, or add an explicit namespace definition.